lib/bundler/audit/scanner.rb

```diff
@@ -68,7 +68,7 @@ def scan(options={},&block)
         ignore = Set[]
         ignore += options[:ignore] if options[:ignore]
 
-        scan_sources(options,&block)
+        scan_sources(options,&block) unless options[:ignore_insecure_sources]
```

Comment:
Wouldn't it be better to still scan and output any problems, but just not consider it a failure?